client/src/boot/BootRoutes.js

```diff
@@ -140,8 +156,64 @@ class BootRoutes {
       });
     });
 
+    window.onbeforeunload = this.handleBeforeUnload;
```

Comment:
I think this is overwriting the `window.onbeforeunload` assigned via LeftAndMain.EditForm.js. See my comment.